sharktank/sharktank/ops/signatures.py

```diff
@@ -1214,6 +1215,25 @@ def _sharded_cat_trampoline(d: SignatureDispatcher, maybe_sharded: AnyTensor):
         d.fail(tensors)
 
 
+@overridable(is_trivially_replicable=False)
+def sharded_gather(input: AnyTensor, root_rank: int) -> AnyTensor:
```

Comment:
```suggestion
def sharded_gather(input: AnyTensor, root_rank: int) -> list[AnyTensor]:
```

Same change for the signature of the trampoline.

Reply:
fixed.